### ISSUE TITLE
TkK findings

### ISSUE THREAD

**Issue by musicEnfanthen** [webern-unibas-ch/awg-app#551]
- [x] M 317 Sk2.1: #path242 needs group element
- [x] M 317 Sk2.1.2: m3, s2, 2/8: Red eighth flag in SVG

**Comment by musicEnfanthen** [comment]
- M 317 Sk4 T. 3B-6:
   - [ ] doubled svgGroupId "g10030" for m. 6, sys. 3-4, and m. 6, sys. 4 ("fis1 durch Einkreisung getilgt"), probably wrong in second case
- M 317 Sk4 T. 7-12: 
  - [ ] Missing tkk entry for tkk-goup m. 12, sys. 8: [xx] ? Or too many classes? (order of svg elements)
  - [ ] Missing tkk entry for tkk-goup m. 13, sys. 15,16: [xx] ? Or too many classes? (order of svg elements)
  - [ ] Missing tkk entry for tkk-goup m. 14, sys. 16: [xx] ? Or too many classes? (order of svg elements)
- M317 Sk3.1.3:
   - [ ] Linkbox grouping for all link boxes missing?
- M317 Sk3.1: 
  - [ ] Nested tkk groups g1225 in g1241 works, but will not be rendered properly
  - [ ] Linkbox grouping for Sk1 missing
- M317 Sk3.1.2:
   - [ ] Linkbox grouping for Sk1 missing
- M317 Sk1:
   - [ ] Linkbox grouping for Sk2.1.3 missing

**Comment by masthom** [comment]
M 317 Sk 4, Missing tkk entry for tkk-group m. 14, sys. 16: 
##looks fine to me. what is wrong?

M317 Sk3.1,  Link to C:Sk1 not correct (in comment for m. {5}, sys. 4, 6/8): 
##looks fine to me, what is wrong?

M317 Sk3.1.3, Linkbox grouping for all link boxes missing?:
##big mystery... grouping exists, sth is wrong anyway. 

for the other findings in M317 see PR #852

**Comment by musicEnfanthen** [comment]
>M 317 Sk 4, Missing tkk entry for tkk-group m. 14, sys. 16:
>##looks fine to me. what is wrong?

The svg displays a tkk-box for "u. von u. 3 sowie Winkel mit rotem Buntstift." which is correct, but another (smaller) tkk-box inside it around the "u. 3". But there is no separate tkk for that location. So wether the tkk comment is missing or there is an additional misplaced tkk-class in the SVG file.

**Comment by masthom** [comment]
> M317 Sk3.1.3, Linkbox grouping for all link boxes missing?:
##big mystery... grouping exists, sth is wrong anyway. 

just a guess: this is the only item so far with linkbox grouping + `"rowtable": true,` might this cause the issue?

**Comment by musicEnfanthen** [comment]
I don't think so. But I recognized that in other files there is a group with an aria-label for the linkBox label within the `g.link-box` , something like:

```
<g
       aria-label="Sk2.1"
       id="text110"
       style="font-weight: 700; font-size: 12px; font-family: Arial-BoldMT, Arial;  isolation: isolate;  fill: #7d7d7d;"
       class="cls-13"> ...
```
This is missing for all link-boxes in Sk3.1.3

**Comment by musicEnfanthen** [comment]
## M30 & M34

- M 34 Sk 1
  - [ ] Missing svgGroupId (and class in svg?) for m. 1, sys. 2, 3/4: Viertelnote gestrichen
 
- M 30 Sk1 
  - T. {9D-12B}
    - [ ] m. `{9D-12B}` in SVG vs. m. `{9D–10D, 11B}, 12B` in sourceDesc

  - T. 7B -{10C}:
    - [ ] `des` = `Des` (?) in: {
                    "svgGroupId": "g1156",
                    "measure": "9C",
                    "system": "11",
                    "position": "1. Note",
                    "comment": "des gestrichen."
                },
  - T. 9E-{15B}
    - [ ]  `{13B–15B}` (?) in: {
                    "svgGroupId": "g1197",
                    "measure": "{13B–14B}",
                    "system": "14–15",
                    "position": "",
                    "comment": "Takte pauschal gestrichen."
                },

**Comment by musicEnfanthen** [comment]
## M212 TF 1

- p. 1
  - [ ] The first tkk in m.1 has as location: `(2/8) bis Taktende` for the hairpin, but the crescendo hairpin ends at 2/4 ?

- p. 2
  - [ ] The tkk for m. 14, 1/8, and  (now) m. 14 1/8 (but probably m. 23, see below) dispaly accidentals in brackets. Correct?
  - [ ] There is no tkk displayed on the very last asterisk m. 23, but a second tkk displayed in m. 14, lower piano, that reads: `14 | Klav. | 1/8 | (♯) zu Gis1/Cis und (♮) zu H/g als redundant weggelassen. B: Akkoladenwechsel nach T. 22.` which is probably the missing tkk for m. 23?

- SourceDescription:
  - [ ] Description mixes `Flecke` and `Flecken` for plural forms.

**Comment by musicEnfanthen** [comment]
## Op. 22 SkRT

- SourceDescription:
  - [ ] Writing Material mentions 3 types of sheets, but refers only to "Bl. 1" and "Bl. 2"
  - [ ] Also: correct that the "shortest" sheet type has 9 systems, and the longer ones have only 8 systems?

- SkRTc
  - [ ] Last tkk on row variant 47 had: `Bl. 3r | 8a | Ub`. Corrected to `Ud` according to SourceDescription and transcription, correct?

**Comment by musicEnfanthen** [comment]
## Op. 22 SkRT

- SkRTa:
  - [ ] Row form Ges should be Kgis (?) in 
`Bl. 1r | 1b | Ges: 10.–11. Reihenton | Auf Rasur: vermutlich ♭b1–♮h1 geändert zu ♭b–♮h.`

- SkRTb:
  - [ ] Tkk for second dividing line is missing (but duplicates svg id and content of that in SkRTa = C&P error?):
`  Bl. 2r | 1b–8b | K- und KU-Formen: 6.–7. Reihenton | Strich  zur Gliederung der Hexachorde mit rotem Buntstift. Vermutlich  nachträglich gezogen, da der Strich mit einzelnen Notenköpfen  kollidiert.` 
  - [ ] Kd should be Ka (?) in 
`Bl. 2r | 1b | Kd | Reihennummerierung 18 mit grünem Buntstift.`

**Comment by musicEnfanthen** [comment]
@chael-mi 

## M 37 Sk1

- SVG:
  - [ ] missing link-box class for linkBox to Sk1

**Comment by musicEnfanthen** [comment]
@chael-mi 

## M 35_42

- SVGs for Sk1.1 and Sk2 missing

- TF1:
  - [ ] T. 14 = 24?
  - [ ] 1. Note = 1/4?

- Sk1b:
  -  [ ] TkK for deleted D in m. {9A} missing

- Sk1d:
  - [ ] 16 = 16B?
  - [ ] TkK for deleted notes in m. {17B} missing
  - [ ] TkK for deletion m. 18 missing
  - [ ] supplied accid classes missing in some cases

- Sk1e:
  -   [ ] 22 = {22}?

- SourceDescription:

**Comment by musicEnfanthen** [comment]
@chael-mi 

## M 22

- Sk1:
  - [ ] m. 2, sys. 23: missing stem for chord?
  - [ ] m. 7, sys. 22, 1.-2. note, commentary: "Oberstimmenschicht: " ?

- Sk2:
  - [ ] m .5m sys. 27, 3/4, commentary: delete "Viertelnote"?

- Sk3:
  - [ ] m. 3, sys. 26, 5/8, commentary: h1/d2 überschreibt a1/c2 [Viertelnote in Notentext ?]
  - [ ] m. 4, sys. 26, 2-3/16: add rest?
  - [ ] m. 5B, sys. 26, 3/8: h2/h1 überschreibt a2?
  - [ ] m. 6, sys. 26, 2/4, commentary: "Unterstimmenschicht: " ?
  - [ ] m. 7, sys. 24:  (5/8) instead of 5/8 ?

- Sk4:
  - [ ]

**Comment by musicEnfanthen** [comment]
@chael-mi 

## M 36

- Sk1:
  - [x] m. 6, sys. 16: 16 = 17? Metrisch falsch gesetzter [r statt n?] <html><body>

"6 | 16 | 3/4 | Metrisch falsch gesetzten Notenkopf e gestrichen."
-- | -- | -- | --

</body>
</html>
  - [x] m. 16, Kommentar fehlt?

**Comment by musicEnfanthen** [comment]
@chael-mi 

## M 38

- Sk2:
  - [ ] Link-Boxes in SVG verweisen auf M 37 Sk2 und Sk3, müsste aber Sk1 und Sk2 sein


- Sk3  
  - [ ] zweites SVG müsste im Titel haben: "T. 5"; hat aber "T. 1-4" wie erstes SVG

**Comment by musicEnfanthen** [comment]
@masthom 

## M 322

- alle SVGs: "supplied"-Klassen fehlen

- Sk3a:
  - [ ] Textcritics zu g9 fehlt.

- Sk3b:
  - [ ] Textcritics zu g9 fehlt.
  - [ ] Textcritics zu g12 fehlt.
  - [ ] Textcritics zu g456 fehlt.
  - [ ] Gruppierung g456 ist im SVG in g22b eingeschlossen.
  - [ ] 
  -

**Comment by masthom** [comment]
g9: bleibt sowohl in Sk3a als auch in Sk3b ohne tkk.
Sk3b, g12: jetzt g1210b. 
Sk3b g456: gehört auch inhaltlich zu g22b, tkk-class war überflüssig. 
Entsprechende Korrekturen folgen mit PR zu M 321.
